docs/design/Makefile

```diff
@@ -1,8 +1,16 @@
 StratisSoftwareDesign.pdf: StratisSoftwareDesign.lyx
 	lyx -batch --export pdf $<
 
+PoolStateDiagrams:
```

Comment:
I think that svg might be a better output file type.

Reply:
Thanks.  Switched to .svg.